gandalf.js

```diff
@@ -1,8 +1,7 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!